src/main/java/seedu/address/ui/body/address/PersonDetailPanel.java

```diff
@@ -86,12 +88,26 @@ private Collection<Label> getTagLabels(Person person) {
     }
 
     private Collection<Region> getDataCardCollection(Person person) {
-        return Stream.of(
+        List<Region> regions = Stream.of(
                 new PersonDetailCard.DetailCardData("Phone", person.getPhone().toString()),
                 new PersonDetailCard.DetailCardData("Address", person.getAddress().toString()),
-                new PersonDetailCard.DetailCardData("Email", person.getEmail().toString()))
+                new PersonDetailCard.DetailCardData("Email", person.getEmail().toString()),
+                new PersonDetailCard.DetailCardData("Gender", person.getGender().toString()),
+                new PersonDetailCard.DetailCardData("Race", person.getRace().toString()),
+                new PersonDetailCard.DetailCardData("Communication channels", person.getComms().toString()),
```

Comment:
Not sure if we want to call this "Preferred communication channel" or just communication channel. Any thoughts?

Reply:
I think if we have a way of marking channels as preferred, then we can add a `Preferred Communication Channels` section.